src/main/scala/firrtl/passes/Passes.scala

```diff
@@ -12,9 +12,21 @@ import firrtl.PrimOps._
 
 import scala.collection.mutable
 
-trait Pass extends LazyLogging {
-  def name: String
+trait Pass extends Transform {
```

Comment:
Would appreciate a comment explaining that Pass is intended to be used as kind of a sub-transform which is why it's inputForm and outputForm are UnknownForm